.shippable.yml

```diff
@@ -56,8 +56,8 @@ build:
     - _make PLATFORM=hikey
     - _make PLATFORM=hikey CFG_ARM64_core=y
     - _make PLATFORM=mediatek-mt8173 CFG_ARM64_core=y
-    - _make PLATFORM=imx-mx6ulevk ARCH=arm CFG_PAGEABLE_ADDR=0 CFG_NS_ENTRY_ADDR=0x80800000 CFG_DT_ADDR=0x83000000 CFG_DT=y DEBUG=y CFG_TEE_CORE_LOG_LEVEL=4
-    - _make PLATFORM=imx-mx6ullevk ARCH=arm CFG_PAGEABLE_ADDR=0 CFG_NS_ENTRY_ADDR=0x80800000 CFG_DT=y DEBUG=y CFG_TEE_CORE_LOG_LEVEL=4
+    - _make PLATFORM=imx-mx6ulevk ARCH=arm
+    - _make PLATFORM=imx-mx6ullevk ARCH=arm
```

Comment:
`ARCH=arm` is useless and should be removed too.

In the commit subject: s/removed/remove/